src/app/GitExtUtils/GitUI/Theming/AppColor.cs

```diff
@@ -22,6 +23,7 @@ public enum AppColor
         GraphBranch6,
         GraphBranch7,
         GraphBranch8,
+        GraphBranchNonRelative,
```

Comment:
Only real branch colors may start with "GraphBranch".
```suggestion
        GraphNonRelativeBranch,
```